Nodejs/Product/TestAdapter/TestFrameworks/Angular/karmaConfig.js

```diff
@@ -19,6 +19,11 @@ module.exports = function (config) {
     // Replace all reporters
     config.reporters = ['vsKarmaReporter'];
 
+    if (!config.hostname && !config.listenAddress) {
```

Comment:
If `hostname` is set but `listenAddress` is not will the discovery fail? 

Shouldn't `listenAddress` be set all the time?

Reply:
Yes it was failing because the default listenAddress is ipv4. Node started using the ipv6 address so this mismatch caused an issue.

Reply:
I mean, with this change if a user sets the `hostname` jbut not the `listenAddress`, the default will never apply to `listenAddress` causing it to fail?

Reply:
Yes, my reasoning was that if the user left the default values for both then their intent was just `localhost` for both. If they changed either value then it was intentional and we won't change the behavior with our options (for example, the hostname could be some server). Not sure if this is a valid assumption..

Reply:
Sounds good, thanks for clarifying.